backend/fms_core/resources/transfer.py

```diff
@@ -0,0 +1,187 @@
+from ._generic import GenericResource
+from ._utils import skip_rows
+from ..models import Container, Process, ProcessSample, Protocol, Sample, SampleKind, SampleLineage
+from ..utils import (
+    VolumeHistoryUpdateType,
+    blank_str_to_none,
+    check_truth_like,
+    create_volume_history,
+    float_to_decimal,
+    get_normalized_str,
+)
+
+
+class TransferResource(GenericResource):
+    source_container = Field(column_name='Source Container Barcode')
+    source_container_coordinates = Field(column_name='Source Location Coord')
+
+
+    destination_container = Field(attribute='destination_container', column_name='Destination Container Barcode',
+                     widget=ForeignKeyWidget(Container, field='barcode'))
+    destination_container_coordinates = Field(attribute='destination_container_coordinates', column_name='Destination Location Coord')
+    destination_container_name = Field(attribute='destination_container_name', column_name='Destination Container Name')
+    destination_container_kind = Field(attribute='destination_container_kind', column_name='Destination Container Kind')
+
+    destination_parent_container = Field(attribute='destination_parent_container', column_name='Destination Parent Container Barcode',
+                      widget=ForeignKeyWidget(Container, field='barcode'))
+    destination_parent_container_coordinates = Field(attribute='destination_parent_container_coordinates', column_name='Destination Parent Container Coord')
+
+    source_depleted = Field(attribute='source_depleted', column_name='Source Depleted')
+    volume_used = Field(column_name='Volume Used (uL)', widget=DecimalWidget())
+    transfer_date = Field(attribute='transfer_date', column_name='Transfer Date', widget=DateWidget())
+    comment = Field(attribute='comment', column_name='Comment')
+
+
+    class Meta:
+        model = Sample
+        import_id_fields = ()
+        fields = (
+            'source_depleted',
+        )
+        excluded = (
+            'container',
+            'sample_kind',
+            'individual',
+            'child_of',
+            'volume_history',
+            'comment',
+        )
+        export_order = (
+            'source_container',
+            'source_container_coordinates',
+            'destination_container',
+            'destination_container_coordinates',
+            'destination_container_name',
+            'destination_container_kind',
+            'destination_parent_container',
+            'destination_parent_container_coordinates',
+            'source_depleted',
+            'volume_used',
+            'transfer_date',
+            'comment',
+        )
+    def before_import(self, dataset, using_transactions, dry_run, **kwargs):
+        skip_rows(dataset, 7)  # Skip preamble
+
+    def import_obj(self, obj, data, dry_run):
+        self.extracted_from = Sample.objects.get(
```

Comment:
Rename this to match what it contains ... self.transfered_from possibly. The current name is just confusing.